docs/docs/contribute/getting-started.md

```diff
@@ -14,109 +14,193 @@ license: CC-BY-SA-4.0
 
 Learn how to set up the development environment to start working with our projects!
 
-## Code Editor ##
 
-A code editor allows you to work with code more easily. We will be using Visual Studio Code (VS Code) for this tutorial. You can download it [here](https://code.visualstudio.com/download), and install it with the default settings.
+## Setting up the C/C++ Compiler
 
-!!! note
-    You are free to use any code editor of your choice, although you will have to complete steps of this tutorial through alternate means.
+A compiler translates human-readable code into machine code. We will be setting up the C/C++ compiler so that our code written in the human-readable C++ language can be understood by the computers!
 
-## Setting up the GCC Compiler ##
+=== "Windows"
 
-A compiler translates human-readable code into machine code. We will be setting up the GCC compiler so that our code written in the human-readable C++ language can be understood by the computers!
+    First, [download Chocolatey](https://docs.chocolatey.org/en-us/choco/setup#installing-chocolatey) and install it using an [administrative shell](https://www.howtogeek.com/194041/how-to-open-the-command-prompt-as-administrator-in-windows-8.1/). Chocolatey is a package manager that simplifies the installation of the tools we will need.
 
-First, [download MSYS2](https://www.msys2.org/) and install it with the default settings. MSYS2 helps to set up MinGW-w64, which uses the GCC compiler to create Windows programs.
+    We will now install the compiler and other build tools using Chocolatey.
+    Open up the Command Prompt and enter the following commands:
 
-We will now install the Mingw-w64 toolset which includes GCC, and other build tools that will be employed later. When the MinGW terminal pops up, enter the following command:
+    ??? tip
+        You can use the Windows Search Bar to find the Command Prompt application.
+        ![Search for the Command Prompt in the Window’s search bar.](assets/search-command-prompt.png)
 
-```
-pacman -Sy --needed --noconfirm base-devel mingw-w64-x86_64-toolchain mingw-w64-x86_64-cmake mingw-w64-x86_64-ninja mingw-w64-x86_64-clang mingw-w64-x86_64-lld wget
-```
+    ```cmd
+    choco install -y cmake --installargs '"ADD_CMAKE_TO_PATH=System"'
+    choco install -y ninja
+    choco install -y llvm
+    refreshenv
+    ```
 
+    !!! note
+        By default, `System` will make `cmake` available to all users on the Windows machine. To only set it for the current user, replace `System` with `User`.
```

Comment:
It's not really by default as system is specified in the code. 

How about this instead?

```The code above will make `cmake` available to all users on the same machine. To only set it for the current user, replace `System` with `User`.```